modules/claims_api/app/swagger/claims_api/v2/production/swagger.json

```diff
@@ -1405,7 +1405,7 @@
                                     "city": {
                                       "description": "City for the Veteran's current mailing address.",
                                       "type": "string",
-                                      "pattern": "^([-a-zA-Z0-9'.#]([-a-zA-Z0-9'.# ])?)+$",
+                                      "pattern": "^$|^([-a-zA-Z'.' ])+$",
```

Comment:
seems like we have a few patterns for the city matching.... is there a reason for the slight discrepancy?
```
^$|^([-a-zA-Z'.' ])+$
^$|([-a-zA-Z'.' ])+$
```

Reply:
No I think I just deleted to far in, updating that now